variable/emissions/emissions.yaml

```diff
@@ -190,6 +190,34 @@ Carbon Sequestration|Land Use|Soil Carbon Management:
                 techniques
    unit: Mt CO2/yr
 
+# Variables related to the CO2 emission by power plant
+Emissions|CO2|Electricity|Biomass|Traditional:
+   description: CO2 emmissions of a typical Traditionnal Biomass power plant
+                biomass from local and pre-existing
+                resources mostly waste :domestic and agricultural waste, forest residues.
+   unit: tons/MWh
+
+Emmissions|CO2|Electricity|Biomass|New and imported :
+   description: CO2 emmissions of a typical new biomass power plant
+                biomass from dedicated energy crops and biomass imports.
+   unit: tons/MWh
+
+Emmissions|CO2|Electricity|Coal:
+   description: CO2 emmissions of a typical coal power plant
+   unit: tons/MWh
+
+Emmissions|CO2|Electricity|Gas|OCGT:
+   description: CO2 emmissions of a typical Open Cycle Gas Turbine power plant
+   unit: tons/MWh
+
+Emmissions|CO2|Electricity|Lignite:
+   description: CO2 emmissions of a typical Lignite power plant
+   unit: tons/MWh
+
+Emmissions|CO2|Electricity|Nuclear:
+   description: CO2 emmissions of a typical nuclear power plant
+   unit: tons/MWh
+
 Emissions|BC:
```

Comment:
Why not use Emissions|Black Carbon to name this variable?

Reply:
Ok, I got it.
Thanks.